src/app.controller.post.ts

```
@@ -79,128 +73,109 @@
-					if (permission.valid && (permission as AuthTablePermissionSuccessResponse).allowed_fields?.length) {
-						if (!queryFields?.length) {
-							queryFields = (permission as AuthTablePermissionSuccessResponse).allowed_fields
-						} else {
-							queryFields.push(...(permission as AuthTablePermissionSuccessResponse).allowed_fields)
-							queryFields = queryFields.filter(field =>
-								(permission as AuthTablePermissionSuccessResponse).allowed_fields.includes(field),
-							)
-						}
-					}
-				}
+		if (!(body instanceof Array)) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)